README.md

```diff
@@ -1,66 +1,55 @@
-## Continuous Load
+# Continuous Load
 
-Continuous Load is a project design to run load 24/7 across an infrastructure and monitor the network health even in production. 
-We have rolled out various versions at clients.
+Continuous Load is a project designed to run load 24/7 across an infrastructure and monitor the network health even in production. 
```

Comment:
One extra like to say the high level goal i.e. "
It's goal is to find problems in infrastructure before the users of that infrastructure find them."

And can we also include an image of the dashboard?